.github/workflows/release.yml

```diff
@@ -62,7 +52,7 @@ jobs:
   publish-s3:
     needs: create-packages
     runs-on: ubuntu-18.04
-    if: github.event_name == 'workflow_dispatch' || startsWith(github.ref, 'refs/tags')
+    if: startsWith(github.ref, 'refs/tags')
     steps:
       - uses: actions/checkout@master
 
```

Comment:
We don't need all those `update_repo.sh` dependencies anymore.

Reply:
Ah, I see, it is in the next commit.

Reply:
Which dependencies? "cd: publish to RWS" commit should remove them all.